Python _Foundation_ (#1304)
temporalio/documentation: docs/python/how-to-register-types-with-a-worker-in-python.md

---
id: how-to-register-types-with-a-worker-in-python
title: How to register types with a Worker in Python
sidebar_label: Register types with a Worker
description: Register types with a Worker
tags:
  - developer-guide
  - sdk
  - python
---

> When a `Worker` is created, it accepts a list of Workflows and/or Activities in the `workflows` and/or `activities` parameters respectively.

Reviewer: ```suggestion
When a `Worker` is created, it accepts a list of Workflows in the `workflows` parameter, a list of Activities in the `activities` parameter, or both.
```
You might be a fan of Cassian Andor, but he has his own TV show.
